en/getting_started/flight_modes.md

```diff
@@ -111,13 +111,14 @@ For multicopters this is the 3D flying mode. You can make flips and acrobatics.
 
 [<img src="../../assets/site/automatic_mode.svg" title="Automatic mode" width="30px" />](#key_automatic)&nbsp;[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](#key_position_fixed)
 
-You can use Hold in case of emergency or during a mission in order to Hold the aircraft at its current position autonomously and pause the flight, while you get time to assess the situation either emergency or mission related. When you activate Hold mode with a pre-programmed switch, the multicopter will fix itself to its current GPS position and altitude and hover there. During hold you are not able to control the craft with the RC. It is the first line of defense if you lose control over the aircraft and panic.
```

Comment:
FYI removed line "It is the first line of defense if you lose control over the aircraft and panic."
1. that depends on your current mode - you might just centre your sticks
2. Captured in the text "help regain control of vehicle"

Reply:
Yep that is good. You can also activate it from QGC (Pause button)